.github/ISSUE_TEMPLATE/simple.yml

```diff
@@ -0,0 +1,48 @@
+name: Other issue (Simple)
+description: Did you encounter technical problems with the build system or something else not covered by other options?
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit an issue!
+        **For *bug reports*, *feature requests* and *documentation issues* please switch to the correct template!**
+
+        Want to ask a question instead? Please don't open an issue!
+        Head to https://ask.openrouteservice.org
+
+
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this? (required)
+      description: Please search to see if an issue already exists for the problem you encountered.
+      options:
+        - label: I have searched the existing issues
+          required: true
+
+  - type: textarea
+    attributes:
+      label: Current behavior (required)
+      description: Clear and concise description of what you are doing and what has happened that doesn't seem right. (required)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected behavior (required)
+      description: Clear and concise description of what you think should have happened instead. (required)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional context
+      description: |
+        Add any further context or screenshots about the problem that might help solve it.
+        Providing context helps us come up with a solution faster.
+
```

Comment:
Apart from the context, if there's problems with the setup, we'd need the following:

* ors config
* setup instructions (are they running docker/docker compose/from source?)
* ors version (sth like last git index or docker image id or sth)
Maybe I'm missing stuff, but that's what came to my mind.

I'd add a **non required**, but pre-filled field with a name along the lines of  "setup information", and some summary sections like in the `response` field of the bug_report.yml

Reply:
I put the info setup info requirements into the placeholder of the context text input in 55aa6d5cd549ade454e7831c10ded6454f560b8f.
That already is optional and we don't need an additional input field for as the simple issue should be kept as simple as possible.